src/core/DatePicker/DatePicker.tsx

```diff
@@ -129,6 +131,11 @@ export type DatePickerProps = {
    * @default false
    */
   showTime?: boolean;
+  /**
+   * Show additional buttons to select year.
+   * @default false
+   */
+  showYear?: boolean;
```

Comment:
I would say to rename maybe to `showYearNavigation` because `showYear` for me sounds like if it is not enabled, you won't see what year it is 😀
Or maybe we should always show it.

Reply:
mb then `showYearSelection` better?

Reply:
renamed to `showYearSelection`